src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -583,25 +569,21 @@ export class MeetingManager implements AudioVideoObserver {
   };
 
   selectVideoInputDevice = async (
-    device: Device | VideoTransformDevice
+    device: Device | VideoTransformDevice | null
   ): Promise<void> => {
-    let receivedDevice = device;
-    if (typeof device === 'string') {
-      receivedDevice = videoInputSelectionToDevice(device);
-    }
-    if (receivedDevice === null) {
+    if (device === null) {
       try {
-        await this.audioVideo?.chooseVideoInputDevice(null);
+        await this.audioVideo?.stopVideoInput();
```

Comment:
I see the use now for `stopVideoInput`, may be I have commented somewhere in `LocalVideoProvider` about its usage.

Reply:
That comment is correct and helps us find a bug in JS SDK.